packages/openfermion/quri_parts/openfermion/ansatz/kupccgsd.py

```diff
@@ -55,53 +60,142 @@ def __init__(
         fermion_qubit_mapping: OpenFermionQubitMapping = jordan_wigner,
         trotter_number: int = 1,
         delta_sz: int = 0,
+        singlet_excitation: bool = False,
```

Comment:
Can you add docstring for `singlet_excitation`?

Reply:
I just added the docstring for it